x/ante/ante_test.go.no

```diff
@@ -92,6 +93,11 @@ type AnteTestSuite struct {
 	txBuilder client.TxBuilder
 }
 
+func SetBalances(ctx sdk.Context, bk bankKeeper.Keeper, addr sdk.AccAddress, coins sdk.Coins) {
```

Comment:
What is the purpose of this ante_test.go.no file, with .no extension?

Reply:
The setup of test for ante requires more work and so I removed go file by renaming it with `.no` extension. It will be fixed in another branch

Reply:
Ok!